docs/guides/Guide-Intro.md

```diff
@@ -55,6 +69,7 @@ const SimpleApp = StackNavigator({
   Home: { screen: HomeScreen },
 });
 
+// if you are using create-react-native-app you don't need this line
```

Comment:
If they're not using `AppRegistery` then they have to `export default` the component. We can just add that into the example above. 

However, I feel like it's confusing to comment out a line when going through a getting started guide. Makes me want to split the tutorials up a bit 🤔 

That shouldn't be a blocker for right now though.

Reply:
FWIW React Native does this in their own documentation: https://facebook.github.io/react-native/docs/tutorial.html#content

Reply:
Ah hadn't realized that! Then the only thing to do here is add that `export default`